frontend/src/context/DialogContext.tsx

```diff
@@ -1,31 +1,25 @@
 import React, { useReducer } from "react";
-
-import { reducer } from "../reducers/dialog.reducer";
 import {
-  OPEN_DELETE_ITEM_DIALOG,
-  CLOSE_DELETE_ITEM_DIALOG,
-  OPEN_DELETE_COLUMN_DIALOG,
-  CLOSE_DELETE_COLUMN_DIALOG,
-  OPEN_EDIT_ITEM_DIALOG,
-  CLOSE_EDIT_ITEM_DIALOG,
-  OPEN_EDIT_COLUMN_DIALOG,
-  CLOSE_EDIT_COLUMN_DIALOG,
-  OPEN_CREATE_ITEM_DIALOG,
-  CLOSE_CREATE_ITEM_DIALOG,
+  CLOSE_CREATE_ITEM_DIALOG, CLOSE_DELETE_COLUMN_DIALOG, CLOSE_DELETE_ITEM_DIALOG, CLOSE_EDIT_COLUMN_DIALOG, CLOSE_EDIT_ITEM_DIALOG, CLOSE_RETRO_ITEM_DETAIL_DIALOG, OPEN_CREATE_ITEM_DIALOG, OPEN_DELETE_COLUMN_DIALOG, OPEN_DELETE_ITEM_DIALOG, OPEN_EDIT_COLUMN_DIALOG, OPEN_EDIT_ITEM_DIALOG, OPEN_RETRO_ITEM_DETAIL_DIALOG
```

Comment:
I'm not sure if the diff is displayed wrong to me, but this code should be formtted.

Reply:
Formatted again.